File: com/pdfjet/Token.java
package com.pdfjet;

import java.nio.charset.StandardCharsets;

public class Token {
    // Fundamental structural tokens
    public static final byte SPACE = (byte) ' ';
    public static final byte NEWLINE = (byte) '\n';
    public static final byte[] BEGIN_DICTIONARY = "<<\n".getBytes(StandardCharsets.US_ASCII);
    public static final byte[] END_DICTIONARY = ">>\n".getBytes(StandardCharsets.US_ASCII);
    public static final byte[] STREAM = "stream\n".getBytes(StandardCharsets.US_ASCII);
    public static final byte[] END_STREAM = "\nendstream\n".getBytes(StandardCharsets.US_ASCII);

    // Object management tokens
    public static final byte[] NEW_OBJ = " 0 obj\n".getBytes(StandardCharsets.US_ASCII);
    public static final byte[] END_OBJ = "endobj\n".getBytes(StandardCharsets.US_ASCII);
    public static final byte[] OBJ_REF = " 0 R\n".getBytes(StandardCharsets.US_ASCII);

    // Text and content tokens
    public static final byte[] BEGIN_TEXT = "BT\n".getBytes(StandardCharsets.US_ASCII);
    public static final byte[] END_TEXT = "ET\n".getBytes(StandardCharsets.US_ASCII);

    // Essential property tokens (used everywhere)
    public static final byte[] LENGTH = "/Length ".getBytes(StandardCharsets.US_ASCII);
    public static final byte[] TYPE = "/Type ".getBytes(StandardCharsets.US_ASCII);
    public static final byte[] RESOURCES = "/Resources ".getBytes(StandardCharsets.US_ASCII);
}

File: net/pdfjet/Token.cs
using System;
using System.Text;

namespace PDFjet.NET {
public class Token {
    // Fundamental structural tokens
    public static readonly byte Space = (byte) ' ';
    public static readonly byte Newline = (byte) '\n';
    public static readonly byte[] BeginDictionary = Encoding.ASCII.GetBytes("<<\n");
    public static readonly byte[] EndDictionary = Encoding.ASCII.GetBytes(">>\n");
    public static readonly byte[] Stream = Encoding.ASCII.GetBytes("stream\n");
    public static readonly byte[] EndStream = Encoding.ASCII.GetBytes("\nendstream\n");

    // Object management tokens
    public static readonly byte[] NewObj = Encoding.ASCII.GetBytes(" 0 obj\n");
    public static readonly byte[] EndObj = Encoding.ASCII.GetBytes("endobj\n");
    public static readonly byte[] ObjRef = Encoding.ASCII.GetBytes(" 0 R\n");

    // Text and content tokens
    public static readonly byte[] BeginText = Encoding.ASCII.GetBytes("BT\n");
    public static readonly byte[] EndText = Encoding.ASCII.GetBytes("ET\n");

    // Essential property tokens (used everywhere)
    public static readonly byte[] Length = Encoding.ASCII.GetBytes("/Length ");
    public static readonly byte[] Type = Encoding.ASCII.GetBytes("/Type ");
    public static readonly byte[] Resources = Encoding.ASCII.GetBytes("/Resources ");
}
}